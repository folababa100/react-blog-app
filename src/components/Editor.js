import React from 'react';

const Editor = () => {
  return (
    <div className="container">
      <div className="row">
        <div className="col-6">
          <form>
            <div className="form-group">
              <input className="form-control" type="text" placeholder="title" />
              <textarea className="form-control" cols="30" rows="10"></textarea>
            </div>
          </form>
        </div>
      </div>
    </div>
  )
}

export default Editor;
