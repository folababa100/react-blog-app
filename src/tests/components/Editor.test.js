import React from 'react';
import { shallow } from "enzyme";
import Editor from '../../components/Editor';

test('should render Editor correctely', () => {
  const wrapper = shallow(<Editor />);
  expect(wrapper).toMatchSnapshot()
})

