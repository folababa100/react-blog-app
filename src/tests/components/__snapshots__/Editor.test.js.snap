// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Editor correctely 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <div
      className="col-6"
    >
      <form>
        <div
          className="form-group"
        >
          <input
            className="form-control"
            placeholder="title"
            type="text"
          />
          <textarea
            className="form-control"
            cols="30"
            rows="10"
          />
        </div>
      </form>
    </div>
  </div>
</div>
`;
